src/transform/src/attribute/analysis.rs

```diff
@@ -0,0 +1,441 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Can we add this to `src/transform/src/analysis.rs` (next to the old `attribute` module)? 

This will make diff of the code that migrates the code a bit easier to maintain.

Reply:
Double checking, but do you mean "move the file to a new path"?

Reply:
Moved to a new location.